nodejs/week1/avg.js

```diff
@@ -0,0 +1,10 @@
+const arguments = process.argv.slice(2);
```

Comment:
arguments could be a reserved javascript variable. could be a good idea to change variable name
https://www.w3schools.com/js/js_reserved.asp

Reply:
Yes it is true. Thanks for the tip.